EventKit/eventkit-cloud
# Update openlayers

# Description of Improvement

Updates openlayers dependency version, fixes tests and typescript errors after version update.

## How to test

Verify map components still appear and work as expected
 - Verify draw tools
 - Verify correct area in box after polygon/box draw
 - Verify extent carried to next steps in create workflow
Verify all tests pass

## Quality Assurance

The following items have been updated:


## Screenshots

:smile:
